submissions/shramkoweb/memory-game/main.js

```diff
@@ -0,0 +1,117 @@
+const HIDE_TIMEOUT = 500;
+const RESET_TIMEOUT = 700;
+const SHOW_ALL_CARDS_TIMEOUT = 1000;
+const CARDS_COUNT_PER_FLIP = 2;
+
+const contentElement = document.querySelector('.content');
+const cellElements = contentElement.querySelectorAll('.cell');
+const resultElement = document.querySelector('.result');
+
+const state = {
+    isBoardBlocked: false,
+    firstCard: null,
+    secondCard: null,
+    cardsLeft: cellElements.length,
```

Comment:
It's not a blocker, but maybe it will be better to rename this property to `cardsCount`.

Reply:
Its not cardsCount because i start from 12 to 0 in game is finish.
So its cards left in board

Reply:
Ok